spec/support/page_objects/trainees/lead_schools_search.rb

```diff
@@ -2,7 +2,7 @@
 
 module PageObjects
   module Trainees
-    class LeadSchoolsSearchPage < PageObjects::Base
+    class LeadSchoolsSearch < PageObjects::Base
```

Comment:
Renamed this for consistency - none of our other page objects have the word `Page` in their name.